doc/usage_recomendation.rst

```diff
@@ -0,0 +1,46 @@
+.. _scenario: options.html#scenario
+.. _PCS: options.html#paramcs
+.. _TAE: tae.html
+
+Usage Recommendation
+===========
```

Comment:
Could you please extend the number of equal characters to be of equal length as the header?